src/hooks/useAdditionalAssetData.ts

```diff
@@ -52,11 +53,13 @@ export default function useAdditionalAssetData(
       uniswapData[address?.toLowerCase()]
   );
 
-  const allTokens = useSelector((state: AppState) => state.uniswap.allTokens);
-  const uniswapToken =
-    allTokens[address?.toLowerCase() === 'eth' ? WETH_ADDRESS : address];
-
-  const totalLiquidity = uniswapToken?.totalLiquidity;
+  const [totalLiquidity, setTotalLiqudity] = useState<number | undefined>();
+  const getTotalLiquidity = useCallback(async () => {
+    const id = address?.toLowerCase() === 'eth' ? WETH_ADDRESS : address;
```

Comment:
just a question here, i see we're using addresses `toLowerCase` everywhere but here `: address`, all good?

Reply:
good call, better safe than sorry